src/components/ItemList.svelte

```diff
@@ -1,4 +1,5 @@
 <script>
+  import * as lunr from "lunr";
```

Comment:
We do this instead of `import lunr from "lunr"` to pass storybook storyshot test.

Reply:
this didn't work, had to revert